### ISSUE TITLE
Add ability to comment expected output.

### ISSUE THREAD

**Issue by marco-a** [marco-a/punit#1]
Right now it's not possible to comment the expected output section.

This would be useful for longer expected outputs:

```
[boolean](true)
[boolean](false)
[boolean](false)
[boolean](true)
```

Here we don't see what's being tested for false / true.

Comments would make this obvious:

```
# condition1 must be true
[boolean](true)
# condition2 & 3 must be false
[boolean](false)
[boolean](false)
# this must be true
[boolean](true)
```

To not break tests that expect `#` somewhere in the output a flag can be added to the punit header:

```
punit,comments
<?php
// ...
```

Which will enable the comment functionality.

**Comment by marco-a** [comment]
Implemented with 185c9ce52f854d9722ab1439450b014fd40a527f.

**Comment by marco-a** [comment]
Maybe add comments to expected vs output diff.
